src/cpp/codegen/call_wrapper.cc

```diff
@@ -0,0 +1,31 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <grpcpp/impl/codegen/call_wrapper.h>
+
+#include <grpcpp/impl/codegen/call_hook.h>
+
+namespace grpc {
+namespace internal {
+
+void Call::PerformOps(CallOpSetInterface* ops) {
+  call_hook_->PerformOpsOnCall(ops, this);
```

Comment:
Why isn't this just in the .h file? It only uses stuff from `impl/codegen` anyway

Reply:
True. This is where I faced the minimal_codegen_test bug and had to add the private virtual to ChannelInterface, but this got left over.
Thanks! Removed